packages/app/src/__tests__/CorePack.test.tsx

```diff
@@ -72,3 +73,61 @@ test('visualize 2D datasets', async () => {
 
   expect(await screen.findByRole('figure', { name: 'twoD' })).toBeVisible();
 });
+
+test('visualize 1D slice of a 3D dataset with and without autoscale', async () => {
+  jest.useFakeTimers('modern');
+  await renderApp();
+  await selectExplorerNode('resilience/slow_slicing');
+
+  // Heatmap is selected by default and fetches a 2D slice.
+  expect(await screen.findByText(/Loading current slice/)).toBeVisible();
+
+  // Let the 2D slice fetch succeed
+  jest.runAllTimers();
+  expect(await screen.findByRole('figure')).toBeVisible();
+
+  // Select the LineVis. The autoscale is on by default: it should fetch a 1D slice.
+  userEvent.click(await screen.findByRole('tab', { name: Vis.Line }));
+  expect(await screen.findByText(/Loading current slice/)).toBeVisible();
+
+  // Let the 1D slice fetch succeed
+  jest.runAllTimers();
+  expect(await screen.findByRole('figure')).toBeVisible();
+
+  // Check that autoscale is truly on
+  const autoScaleBtn = await screen.findByRole('button', {
+    name: 'Auto-scale',
+  });
+  expect(autoScaleBtn).toHaveAttribute('aria-pressed', 'true');
+
+  // Move to other slice to fetch new slice
+  // eslint-disable-next-line prefer-destructuring
+  const d0Slider = screen.getAllByRole('slider', {
+    name: 'Dimension slider',
+  })[0];
+  d0Slider.focus();
+  userEvent.keyboard('{PageUp}');
+  expect(await screen.findByText(/Loading current slice/)).toBeVisible();
+
+  // Let the new slice fetch succeed
+  jest.runAllTimers();
+  expect(await screen.findByRole('figure')).toBeVisible();
+  d0Slider.blur(); // remove focus to avoid state update after unmount
```

Comment:
I don't think you need this one, since you're focusing something else later. You can also use `{ArrowUp/Down}` instead of `{PageUp/Down}` to simplify things.